src/components/header/selectMenuStyles.js

```diff
@@ -0,0 +1,56 @@
+const selectMenuStyles = (selectedTeamColor) => {
+  return (
+      {
+          container: (provided) => ({
+              ...provided,
+              backgroundColor: `${selectedTeamColor}`,
+              height: '100%',
+              cursor: 'pointer',
+              width: '305px',
+          }),
+          control: () => ({
+              display: 'flex',
+              height: '100%',
+          }),
+          valueContainer: (provided) => ({
+              ...provided,
+               width: '285px',
+              padding: '0',
+              margin: '0',
+              color: 'white',
+              fontFamily: 'spurs',
+              fontSize: '22px',
+          }),
+          singleValue: (provided) => ({
+              ...provided,
+              color: 'white',
+              margin: '0',
+          }),
+          indicatorsContainer: (provided) => ({
+              ...provided,
+              display: 'flex',
+              justifyContent: 'center',
+              width: '30px',
+              paddingRight: '22px',
```

Comment:
For a lot of these numerical values, you don't need to make them strings. Easier to write as numbers.

Example:
```
width: 30,
```

change throughout this file where applicable